packages/components/pie-input/test/visual/pie-input.spec.ts

```diff
@@ -1,15 +1,60 @@
-
 import { test } from '@sand4rt/experimental-ct-web';
 import percySnapshot from '@percy/playwright';
+import type {
+    PropObject, WebComponentPropValues, WebComponentTestInput,
+} from '@justeattakeaway/pie-webc-testing/src/helpers/defs.ts';
+import {
+    getAllPropCombinations, splitCombinationsByPropertyValue,
+} from '@justeattakeaway/pie-webc-testing/src/helpers/get-all-prop-combos.ts';
+import {
+    createTestWebComponent,
+} from '@justeattakeaway/pie-webc-testing/src/helpers/rendering.ts';
+import {
+    WebComponentTestWrapper,
+} from '@justeattakeaway/pie-webc-testing/src/helpers/components/web-component-test-wrapper/WebComponentTestWrapper.ts';
 import { percyWidths } from '@justeattakeaway/pie-webc-testing/src/percy/breakpoints.ts';
-import { PieInput, InputProps } from '../../src/index.ts';
+import { statusTypes } from '../../src/defs.ts';
+import { PieInput } from '../../src/index.ts';
+
+const props: PropObject = {
+    status: statusTypes,
+};
+
+// Renders a <pie-input> HTML string with the given prop values
+const renderTestPieInput = (propVals: WebComponentPropValues) => `<pie-input status="${propVals.status}"></pie-input>`;
+
+const componentPropsMatrix: WebComponentPropValues[] = getAllPropCombinations(props);
+const componentPropsMatrixByStatus: Record<string, WebComponentPropValues[]> = splitCombinationsByPropertyValue(componentPropsMatrix, 'status');
+const componentStatus: string[] = Object.keys(componentPropsMatrixByStatus);
 
-test.describe('PieInput - Visual tests`', () => {
-    test('should display the PieInput component successfully', async ({ page, mount }) => {
-        await mount(PieInput, {
-            props: {} as InputProps,
-        });
+test.beforeEach(async ({ mount }, testInfo) => {
+    testInfo.setTimeout(testInfo.timeout + 40000);
```

Comment:
Was the `+ 40000` a temporary fix for something?

Reply:
it's a temporary fix that used in8 files in the codebase 🙄 
maybe @JoshuaNg2332 or @siggerzz has any idea?